examples/basic-host/src/implementation.ts

```diff
@@ -45,6 +45,13 @@ interface UiResourceData {
   csp?: {
     connectDomains?: string[];
     resourceDomains?: string[];
+    frameDomains?: string[];
+    baseUriDomains?: string[];
+  };
+  permissions?: {
+    camera?: boolean;
```

Comment:
let's make these objects (as in capabilities) for future extensions?
e.g. what if one day there's fine-grained vers. coarse geolocation, or front vs. back camera permission, etc.

Reply:
Like CSP, Permissions are coupled to the browser spec ([Permissions Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Permissions_Policy)). I don't think we should diverge at this point. WDYT?